src/java.security.jgss/share/classes/sun/security/jgss/wrapper/SunNativeProvider.java

```diff
@@ -71,8 +71,9 @@ static void debug(String message) {
                         DEBUG = Boolean.parseBoolean(
                             System.getProperty("sun.security.nativegss.debug"));
                         try {
+                            Class.forName("java.net.InetAddress");
```

Comment:
The returned value of Class.forName() is not used, someone else could delete this line later is he/she does not know the story behind this line.  Would you mind add a few line comments about why it is added?

Otherwise, looks good to me.

Reply:
Definitely. I should have added one. Thanks for reminding.